src/Fixer/PhpUnit/PhpUnitTestClassRequiresCoversFixer.php

```diff
@@ -76,12 +76,14 @@ protected function applyPhpUnitClassFix(Tokens $tokens, int $startIndex, int $en
                 'covers',
                 'coversDefaultClass',
                 'coversNothing',
+                'coversTrait',
```

Comment:
Was there ever an annotation `@coversTrait` in PHPUnit?

Reply:
I dug in the docs and couldn't find it. Looked at 9.5 and 10.5. 

I thought it existed but probably did `@covers('Trait')`